intellij-ide.md

```diff
@@ -110,7 +110,30 @@ To edit your code and hot reload changes:
 1. Do not press the 'Stop' button; rather, to see your changes just click the
 **Hot Reload button** (the button with the lightning bolt icon).
 
-### Advanced debugging with Observatory
+## Advanced debugging
+
+### Debugging visual layout issues
+
+To debug a visual issue, start the app with 'Debug', and then open the Flutter tool window using 'View > Tool Windows > Flutter'. This offers three tools for visual debugging:
```

Comment:
We have additional actions available under the gear menu, to the right of the actions. We should doc these as well (and we may want to re-think how they're exposed).

<img width="351" alt="screen shot 2017-07-05 at 7 07 26 am" src="https://user-images.githubusercontent.com/1269969/27868199-dcb74e64-6150-11e7-9a63-3e44f65578b0.png">